sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -171,6 +171,24 @@ private async Task ProcessUntilCanceled(CancellationToken cancellationToken)
 }
 ```
 
+### Authenticating with Azure.Identity
+
+The [Azure Identity library](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity/README.md) provides easy Azure Active Directory support for authentication.
+
+```csharp
+var storageConnectionString = "<< CONNECTION STRING FOR THE STORAGE ACCOUNT >>";
+var blobContainerName = "<< NAME OF THE BLOBS CONTAINER >>";
+var fullyQualifiedNamespace = "<< FULLY QUALIFED EVENT HUBS NAMESPACE >>";
+var eventHubName = "<< NAME OF THE EVENT HUB >>";
+var consumerGroup = "<< NAME OF THE EVENT HUB CONSUMER GROUP >>";
+
+var storageClient = new BlobContainerClient(storageConnectionString, blobContainerName);
```

Comment:
Seems a bit odd that we'd use AAD for authorization with Event Hubs but a connection string for Storage.   Should we consider having them both use the identity credential?

Reply:
> Should we consider having them both use the identity credential?

Works for me.  The reason I didn't do this initially was that we'd also have to note that you'll need to assign some additional Storage related roles to your principal (since the data plane operations are guarded by additional permissions).  If you think that's net less confusing (maybe we can just link to https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/README.md#authenticating-with-azureidentity and be a little more explicit there about the roles you might want to assign (they are described on one of the linked pages)

Reply:
I ended up fleshing things out in the Event Hubs readme, for locality... but it probably does make sense to expand that area of the storage readme as well.   If you're speaking with them already, you may also want to call out that they seem to be conflating `authorization` and `authentication` in there as well.    Azure Identity is performing the authentication, their snippet demonstrates using that credential for authorization.